kitsune/search/v2/documents.py

```diff
@@ -90,68 +89,52 @@ def get_model(cls):
         return wiki_models.Document
 
 
-class QuestionDocument(DSLDocument):
+class QuestionDocument(SumoDocument):
     """
     """
 
-    question_id = field.Integer()
-    question_title = field.Keyword(fields={"text": field.Text()})
```

Comment:
maybe it still makes sense to keep both this as keyword and localized text. We can do this in another PR but let's make sure it's captured in an issue.

Reply:
I don't think keyword is a good fit here - it'll require the search term to exactly match the title